package.json

```diff
@@ -83,11 +83,12 @@
     "typedoc": "^0.11.1"
   },
   "dependencies": {
+    "@types/node": "^12.7.8",
```

Comment:
Doubt this needs to be a dependency. Maybe a `devDependency`.

Reply:
yes you are right this should be a devDependency.
And @graywolf336 how do i test this for a rocket.chat app can you help me with that i am kinda new to this

Reply:
We have updated the README of this project with details on how to test it. So you will need to develop an App or download apps to test and ensure everything runs as expected.

Reply:
ok thanks working on it